src/components/common/SelectField.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { UseFormRegisterReturn } from 'react-hook-form';
+
+interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+  label: string;
+  error?: string;
+  id: string;
+  register?: UseFormRegisterReturn;
+}
+
+const SelectField: React.FC<SelectProps> = ({
+  label,
+  register,
+  error,
+  id,
+  ...props
+}) => (
+  <div className="selectWrapper">
+    <label htmlFor={id} className={error ? 'error' : ''}>
+      {label}
+    </label>
+    <select className={error ? 'error' : ''} id={id} {...props} {...register} />
+    {error ? <span>{error}</span> : <span />}
```

Comment:
Do You need an empty span here? If not just use && instead of ?

Reply:
I need it, it avoids some resizing and spacing issues (input fields components works in the same way) :)

Reply:
Right!